packages/devextreme/testing/tests/DevExpress.ui.widgets/tabs.tests.js

```diff
@@ -422,6 +422,100 @@ QUnit.module('General', () => {
 });
 
 QUnit.module('Tab select action', () => {
+    QUnit.module('onSelectionChanging', {
+        beforeEach() {
+            this.onSelectionChangingSpy = sinon.spy();
+            this.onSelectionChangedSpy = sinon.spy();
+            this.clock = sinon.useFakeTimers();
+
+            this.$tabs = $('#tabs').dxTabs({
+                items: [
+                    { text: '0' },
+                    { text: '1' },
+                    { text: '2' },
+                    { text: '3' }
+                ],
+                onSelectionChanging: this.onSelectionChangingSpy,
+                onSelectionChanged: this.onSelectionChangedSpy,
+                selectedIndex: 0
+            });
+            this.tabInstance = this.$tabs.dxTabs('instance');
+        },
+        afterEach() {
+            this.clock.restore();
+        }
+    }, () => {
+        QUnit.test('cancelling selection event synchronously', function(assert) {
```

Comment:
[test naming]
Kindly rename all tests using a pattern which we usually rely on: "what should happen / how should it work in this case"
```suggestion
        QUnit.test('selection should be cancelled if cancel parameter is set to true', function(assert) {
```

Reply:
[bebf086](https://github.com/DevExpress/DevExtreme/pull/27860/commits/bebf0867b9c1214b545d0d2fb3c7c46c2e3ff9c8)